apps/mdn/utils/percona_toolkit/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# https://github.com/fr3nd/docker-percona-toolkit/blob/master/Dockerfile
+FROM debian:jessie-slim
```

Comment:
Debian 8 (jessie) is the old stable release, supported as LTS until [June 2020](https://wiki.debian.org/LTS), Debian 9 (stretch) is the current stable release, and will be supported until 2022.

Do we want to switch to stretch? How hard would it be?

Reply:
I'll switch to stretch